.github/workflows/container.yml

```
@@ -221,8 +221,8 @@ jobs:
             ${{ inputs.tachidesk_release_type == 'stable' && 'ghcr.io/suwayomi/suwayomi-server:latest' || '' }}
             ghcr.io/suwayomi/suwayomi-server:${{ inputs.tachidesk_release_type }}
             ghcr.io/suwayomi/suwayomi-server:${{ steps.get_latest_release_metadata.outputs.release_tag }}
-        env:
-          SOURCE_DATE_EPOCH: 0
+          cache-from: type=registry,ref=ghcr.io/suwayomi/tachidesk:buildcache
+          cache-to: type=registry,ref=ghcr.io/suwayomi/tachidesk:buildcache,mode=max
```

`tachidesk` label is legacy
```suggestion
          cache-from: type=registry,ref=ghcr.io/suwayomi/suwayomi-server:buildcache
          cache-to: type=registry,ref=ghcr.io/suwayomi/suwayomi-server:buildcache,mode=max
```